### DISCUSSION TITLE
transpilation of node modules

### DISCUSSION THREAD

**Discussion by ntraut** [quasarframework/quasar#9896] [General - Components / Directives / etc] [answered]
Hello how can I set up the transpilation of node modules?
I have this browser list in my package.json:
```
  "browserslist": [
    "defaults",
    "ie >= 11"
  ]
```
and my `babel.config.js` is this one:
```
module.exports = {
  presets: [
    [
      '@quasar/babel-preset-app'
    ]
  ],
  plugins: [
    [
      'module-resolver', {
        'alias': {
          '~': './src'
        }
      }
    ]
  ]
}
```
But for example I have jimp as a dependency and it is not transpiled after `quasar build`: I have arrows functions (with `=>`) in `dist/spa/js/vendor.033ab64b.js`. Is there a config to add to avoid that?
I'm using quasar v1 and @quasar/app v2.

**Comment by ontwikkelfabriek** [comment, marked as answer]
@ntraut 

Use the `transpileDependencies` property: 
https://v1.quasar.dev/quasar-cli/quasar-conf-js#property-build

**Reply by ntraut** [reply]
Thank you, that's exactly what I was looking for, but with all those dependencies, it's not easy to find out which ones need to be transpiled, the only practical solution is maybe to transpile all of them.
